checker/manual/fenum-checker.tex

```diff
@@ -64,15 +64,13 @@ \section{Fake enum annotations\label{fenum-annotations}}
 file \code{\emph{MyFenum}.java}:
 
 \begin{alltt}
-package \textit{myproject}.qual;
```

Comment:
The passages later asks them to pass the full path to myproject to -AqualDirs, thus the class loader would start looking for classes within this folder. If the qualifier is in package myproject.qual, then the folder that needs to be passed to -AqualDirs is the parent folder of myproject.

Either naming approach is fine, what would be preferred?